compiler/rustc_interface/src/tests.rs

```diff
@@ -647,6 +649,9 @@ fn test_unstable_options_tracking_hash() {
     untracked!(dump_mir_dir, String::from("abc"));
     untracked!(dump_mir_exclude_pass_number, true);
     untracked!(dump_mir_graphviz, true);
+    untracked!(dump_mir_spanview, Some(MirSpanview::Statement));
+    untracked!(dump_mono_stats, SwitchWithOptPath::Enabled(Some("my_crate-mono-items.md".into())));
```

Comment:
Just to avoid confusion with the extension, this should be a directory here, rather than a file-looking path.